cli/types/index.d.ts

```diff
@@ -1069,24 +1069,36 @@ declare namespace Cypress {
      */
     hash(options?: Partial<Loggable & Timeoutable>): Chainable<string>
 
+    /**
+     * Invoke a function on the previously yielded subject.
+     *
+     * @see https://on.cypress.io/invoke
+     */
+    invoke<K extends keyof Subject, F extends ((...args: any[]) => any) & Subject[K], R = ReturnType<F>>(
+      functionName: K,
+      ...args: any[]
+    ): Chainable<R>
+    invoke<K extends keyof Subject, F extends ((...args: any[]) => any) & Subject[K], R = ReturnType<F>>(
+      options: Loggable,
+      functionName: K,
+      ...args: any[]
+    ): Chainable<R>
+
     /**
      * Invoke a function in an array of functions.
      * @see https://on.cypress.io/invoke
      */
     invoke<T extends (...args: any[]) => any, Subject extends T[]>(index: number): Chainable<ReturnType<T>>
     invoke<T extends (...args: any[]) => any, Subject extends T[]>(options: Loggable, index: number): Chainable<ReturnType<T>>

```

Comment:
these are redundant now, right?

Reply:
I believe these signatures are capturing a slightly different case than the others (i.e., index-access into array of functions, which lets us type the return value somewhat more narrowly; note we're specifying that `Subject extends T[]` which isn't quite like the other forms?)